stacks/argocd/render.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
this file can be deleted along w/ stacks/argocd/yaml/argocd.yaml

Reply:
+1